linux_os/guide/system/software/integrity/fips/package_dracut-fips_installed/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: rhel6,rhel7,rhel8,fedora
+prodtype: rhel6,rhel7
```

Comment:
This rule is still applicable to rhel8. Please undo the changes.

Reply:
@redhatrises Why do you think this rule is applicable to RHEL 8? Package `dracut-fips` doesn't exist in RHEL 8.

Reply:
@jan-cerny ooppss... you are right... I checked the wrong RHEL8 build. Nevermind on this rule.